src/_guides/language/effective-dart/style.md

```diff
@@ -228,6 +228,24 @@ Db
 {% endprettify %}
 
 
+### DON’T use prefix letters to indicate type, mutability, or scope.
```

Comment:
I worry this is too specific and readers may assume it is OK to use prefixes for other things. How about:

```
### DON'T decorate identifiers with letters that describe the kind of thing being named.
```

?

Reply:
I'd like to be a little less general and use fewer words, if possible. Let's brainstorm offline...